composer.json

```diff
@@ -33,7 +33,7 @@
         "drupal/content_moderation_notifications": "^3.4",
         "drupal/core-composer-scaffold": "9.5.9",
         "drupal/core-project-message": "9.5.9",
-        "drupal/core-recommended": "9.5.9",
+        "drupal/core-recommended": "9.5.10",
```

Comment:
As long as we're upgrading this to 9.5.10, I think it makes sense to upgrade the rest of drupal to 9.5.10. According to https://www.drupal.org/project/drupal/releases/9.5.10 it's a bug-fix release. I ran `bin/composer require drupal/core-recommended:9.5.10 drupal/core-composer-scaffold:9.5.10 drupal/core-project-message:9.5.10 --update-with-all-dependencies` (as recommended there, but with "bin/" and that seems like the way to go.